configure.ac

```diff
@@ -339,36 +339,29 @@ EXTRA_WARNINGS=""
 
 WARNLIST="
 	all
+	extra
```

Comment:
-Wextra includes lots of warnings that aren't in -Wall. Some of them were already in place, I've removed the ones that were subsumed.